airflow/providers/apache/hdfs/hooks/webhdfs.py

```diff
@@ -93,12 +93,19 @@ def _find_valid_server(self) -> Any:
 
     def _get_client(self, connection: Connection) -> Any:
         connection_str = f'http://{connection.host}:{connection.port}'
+        session = None
+
+        if connection.extra_dejson.get('SSL', False):
```

Comment:
small NIT. Can we please change it to "use_ssl" instead of 'SSL' ?  We are mostly using lowercase for extra names.

Reply:
changed